lib/resources/content/cypress.json

```diff
@@ -0,0 +1,9 @@
+{
+  "baseUrl": "http://localhost:8080",
```

Comment:
8080 is the default port for our webpack setup. But cli bundler uses default port 9000. Need to separate template for webpack and cli bundler.

Reply:
FYI, protractor conf is js file, it reads the port from aurelia.json.
```js
const aureliaConfig = require('../aurelia_project/aurelia.json');
const port = aureliaConfig.platform.port;
```

Reply:
Good point, I completely overlooked this configuration. Will change asap

Reply:
Fixed, moved config into JS file and use via aurelia task. Cypress does not support a js config out of the box but I am able to pass in the configuration when running Cypress.